src/main.rs

```diff
@@ -142,6 +149,9 @@ fn init() -> Result<Config> {
              .long("attributes")
              .help("The file into which the metadata attributes are written")
              .takes_value(true))
+        .arg(Arg::with_name("check-in")
```

Comment:
I picked this initially as a random name, but I still had some minor doubts in my mind:
 1) is `--check-in` a good flag choice for this?
 2) in case we have some cloud platform in the future with a user-parameter, should this option take an argument too?

Reply:
> is `--check-in` a good flag choice for this?

I personally like it. It's short and descriptive. Another possibility is `--phone-home` maybe? It matches the Packet terminology and might make it more obvious it's about *sending* data (whereas the other switches are about retrieving data).

> in case we have some cloud platform in the future with a user-parameter, should this option take an argument too?

The parameter would have to be specified through ct/Ignition, right? In which case, it'd probably be cleaner to pick it up from from an env var or a file instead? If it really comes down to adding CLI support for a parameter, I think I'd rather make it a new switch to be explicit, e.g. `--check-in-arg=..` or something.

Reply:
Yes, it would be an additional parameter, I was thinking about the case where we lift it from `EnvinronmentFile` to the CLI, and whether to make this already future-proof for that.

Thanks for the feedback on both, it looks like the status quo is fine then.